### DISCUSSION TITLE
JQuery 3.6.0 vulnerability

### DISCUSSION THREAD

**Discussion by Charismatic-Megafauna** [jquery/jquery#5062] [Show and tell]
I don't know if this has already been mentioned.  I cannot find it but if it has apologies for the duplication
In my company JQuery 3.6.0 was flagged by BlackDuck.  I removed the comments from our code in the same way as https://github.com/jquery/jquery/commit/e24f2dcf3f6bda1a672502e0233c732065cbbe89.
The scan cleared and the issue was closed
Different companies my use different scans and your mileage may vary but its worth a shot

**Comment by Maheswaran-SE** [comment]
Hi, 

Blackduck has reported that the version 3.6.0 has the following vulnerability.  Can you please let me know when you could release version with the following issue is addressed ?

"jQuery contains commented references to the hijacked domain blindsignals, within the files src/queue/delay.js and test/data/jquery-1.9.1.js (the former referring to a Web Archive version of the original site). Users without awareness of the domain's status could be exposed to unspecified attacks if they attempt to follow the links to the hijacked site."

Thanks !

**Comment by mgol** [comment]
jQuery 3.6.1 with the fix is out now: https://blog.jquery.com/2022/08/26/jquery-3-6-1-maintenance-release/
